simulation.md

```diff
@@ -0,0 +1,454 @@
+---
+layout: page
+title: Second analysis steps
+subtitle: Generating signal decays
+minutes: 60
+---
+
+> ## Learning Objectives {.objectives}
+>
+> * Understand how a signal decay sample is produced in the LHCb framework
+> * Produce generator level Monte Carlo, print the decay tree and produce nTuples
+> * Read a DecFile and understand what it produces, including generator level cuts
+> * Filter a simulated sample to reduce disk space needed
+
+## What is Gauss?
+
+The LHCb simulation framework which steers the creation of simulated events and interfaces to multiple external applications. Most commonly, an event is created via the following procedure:
+
+1. The `ProductionTool` (Pythia, GenXicc, ...) generates an event with the required signal particle. Either by generating minimum bias events until a matching particle is found or by ensuring one is produced in every event.
+2. The signal particle is decayed using the `DecayTool` (EvtGen) to the desired final state, all remaining unstable particles are decayed independently.
+3. The signal and its decay products might be required to pass generator level cuts implemented as a `CutTool`.
+4. Particles are transported through the detector simulation.
```

Comment:
I found the description in [LHCBGAUSS-1090](https://its.cern.ch/jira/browse/LHCBGAUSS-1090) to be quite interesting, because it describes the logic that decides which particles are decayed by EvtGen and which are decayed by Geant4.

> * if a particle is known to EvtGen, then it is declared stable in the production generator – it will be decayed by EvtGen
> * if a particle is not known to EvtGen, then it is decayed in the production generator
> * if a particle is not known to EvtGen or the production generator, it will be decayed by Geant4

Maybe it would be interesting to include something like this?